src/telemetry/telemetryService.ts

```diff
@@ -108,6 +108,13 @@ type ParticipantResponseFailedProperties = {
   error_name: ParticipantErrorTypes;
 };
 
+export type ParticipantPromptProperties = {
+  command: string;
+  user_input_length: number;
+  total_message_length: number;
+  has_sample_documents: boolean;
```

Comment:
Should we also track the amount of messages in the chat? Could be useful to see how much people start new chats vs keep existing ones going.

Reply:
Added `history_length` for this. Note that this only tracks the number of history messages and not the length of the messages themselves.